api_tests/metrics/test_registery_moderation_metrics.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from waffle.testutils import override_switch
+
+import time
+from osf import features
+from osf_tests.factories import RegistrationFactory
+from osf.utils.workflows import RegistrationModerationStates, RegistrationModerationTriggers
+from osf.metrics import RegistriesModerationMetrics
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.django_db
+class TestRegistrationModerationMetrics:
+
+    @pytest.fixture()
+    def registration(self):
+        return RegistrationFactory()
+
+    @pytest.fixture(autouse=True)
+    def enable_elasticsearch_metrics(self):
+        with override_switch(features.ELASTICSEARCH_METRICS, active=True):
+            yield
+
+    @pytest.mark.es
+    def test_record_transitions(self, registration):
```

Comment:
Honest question: Since we're using different fields than other metrics, is it worth testing some of the aggregation functions? or do we trust those implicitly based on tests elsewhere?

Reply:
How the data will be aggregated isn't yet know, probably it will only be queried by our stats people via the `/raw` metrics endpoint and not really used internally via the OSF.

Reply:
The `/metrics/raw` endpoint isn't meant for being used on production; it was added so that we could do debugging on staging environments. I think we'd need a separate endpoint for querying these metrics.

Reply:
@brianjgeiger I'm curious why don't we use it for production? It has permissions to prevent any random user from accessing the info.

Reply:
Because we're trying to get away from arbitrary access to our elastic servers, whether by us or others.

Reply:
@brianjgeiger I've now added a read-only endpoint that just runs one per-defined query. You can see the example output in the views comments.